extension/history.js

```diff
@@ -1,10 +1,11 @@
 /* globals log */
 
 export class Database {
+  static readonly = "readonly";
+  static readwrite = "readwrite";
```

Comment:
And this probably has to be changed to `Database.prototype.readonly = "readonly";`